NeoML/include/NeoML/Dnn/Dnn.h

```diff
@@ -662,6 +663,8 @@ class NEOML_API CDnn : public CDnnLayerGraph {
 	void reshape();
 	void rebuild();
 	size_t getOutputBlobsSize() const;
+	const CDnn& getOwnerDnn() const
+	{ return ( owner == nullptr || owner->GetDnn() == nullptr ) ? ( *this ) : ( owner->GetDnn()->getOwnerDnn() ); }
```

Comment:
Honestly, I really do not want to shift or to remove braces
For me now it is most readable

Reply:
@AndrewAndrianov Should we follow the company codestyle here? If so, I can only suggest making a PR to FineObj/CodeStyle

Reply:
I think we should use our codestyle everywhere in our code including NeoML code.
The code of common libraries should set the standard of quality and serve as an example for all developers.

there is special rule in CodeStyle 4.1:
"If a one-liner method is long, its implementation can be written on a separate line with indentation."

Rule 11.6 says:
"The `? :` operation should be written with spaces around `?` and `:`. In case of long conditions and operands, each should be written on a new line."

Reply:
fixed